cli/popper/gha.py

```diff
@@ -254,11 +254,17 @@ def run(self, action, skip_clone, skip_pull, skip, workspace,
         WorkflowRunner.instantiate_runners(
             runtime, new_wf, workspace, dry_run, skip_pull, self.wid)
 
+        # If a runtime configuration file is passed through the
+        # --engine-conf option, parse it and pass it to ActionRunner(s).
+        engine_config = pu.parse_engine_configuration(engine_conf)
```

Comment:
can this be passed to the `instantiate_runners` function instead? then the config dictionary can be given to the ActionRunner constructor and reference to it as a class property. This helps to avoid having to pass the configuration around all the internal methods of runners (e.g. `run`, `docker_run`, etc.)

Reply:
Yeah, that sounds much better. Thanks.